Makefile

```diff
@@ -54,8 +54,10 @@ endif
 #This rule need to be run on RHEL with podman installed.
 container_rhel: build
 	cp $(OUT_DIR)/$(ARCH)/$(BINARY_NAME) $(TEMP_DIR)
+	cp LICENSE $(TEMP_DIR)/license.txt
```

Comment:
@srinivas-kandula Just a note, don't use the master branch to build the container image for v1.0.3. Create a branch based on the released tag v1.0.3. 

I recently updated the Makefile version to 1.0.4 (for a future version). And the master branch also includes some recent changes (related to events) that are not yet released.

Reply:
For RH certification will build collector from tag v1.0.3.